src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -130,11 +130,22 @@ async function initializeConnection(options: Required<BlazorOptions>, circuitHan
   }
 
   DotNet.attachDispatcher({
-    beginInvokeDotNetFromJS: (callId, assemblyName, methodIdentifier, dotNetObjectId, argsJson) => {
-      connection.send('BeginInvokeDotNetFromJS', callId ? callId.toString() : null, assemblyName, methodIdentifier, dotNetObjectId || 0, argsJson);
-    },
+    beginInvokeDotNetFromJS: serverDispatcher,
```

Comment:
This seems like a really dirty hack. Ideally these things would have first class APIs that work differently depending on the platform.

Reply:
I got rid of this in favor of a first class concept. The hackyness was to validate the approach.